entity-framework/core/get-started/aspnetcore/new-db.md

```diff
@@ -109,7 +109,7 @@ A production app would typically put each class in a separate file. For the sake
 
 ## Register the context with dependency injection
 
-Services (such as `BloggingContext`) are registered with [dependency injection](http://docs.asp.net/en/latest/fundamentals/dependency-injection.html) during application startup. Components that require these services (such as MVC controllers) are provided these services via constructor parameters or properties.
+Services (such as `BloggingContext`) are registered with [dependency injection](http://docs.asp.net/en/latest/fundamentals/dependency-injection.html) during application startup. Components that require these services (such as MVC controllers) are provided via constructor parameters or properties.
```

Comment:
What about this?

> Services (such as `BloggingContext`) are registered with [dependency injection](http://docs.asp.net/en/latest/fundamentals/dependency-injection.html) during application startup so that they can be provided automatically to components that consume services (such as MVC controllers) via constructor parameters and properties.

Reply:
Also make this the first sentence of the topic:

> To make `BloggingContext` available to MVC controllers, register it as a service.

The rest is just a generic explanation of the role of DI.